src/ViewModel/Converter/AnnualReportTeaserConverter.php

```diff
@@ -3,20 +3,40 @@
 namespace eLife\Journal\ViewModel\Converter;
 
 use eLife\ApiSdk\Model\AnnualReport;
+use eLife\Journal\Helper\DownloadLinkUriGenerator;
+use eLife\Patterns\PatternRenderer;
 use eLife\Patterns\ViewModel;
 use eLife\Patterns\ViewModel\Teaser;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
 final class AnnualReportTeaserConverter implements ViewModelConverter
 {
+
+    private $patternRenderer;
+    private $downloadLinkUriGenerator;
+
+    public function __construct(PatternRenderer $patternRenderer)
+    {
+        $this->patternRenderer = $patternRenderer;
+    }
+
     /**
      * @param AnnualReport $object
      */
     public function convert($object, string $viewModel = null, array $context = []) : ViewModel
     {
+        $links = [
+            "<a href=\"{$object->getUri()}\">View report</a>",
+        ];
+
+        if ($object->getPdf()) {
+            $links[] = "<a href=\"{$object->getPdf()}\">Download PDF</a>";
```

Comment:
I see you've spotted the `DownloadLinkUriGenerator`, we should use it here to create a download URI for the PDF (so it actually downloads, rather than just opens in the browser).